buildSrc/src/main/kotlin/Config.kt

```diff
@@ -11,11 +11,11 @@ object Config {
     }
 
     object Plugins {
-        const val android = "com.android.tools.build:gradle:3.2.0-alpha15"
```

Comment:
I didn't even notice that we had moved to `alpha` in the gradle --> kotlin switch.  Is there anything more stable that will work for us?

Reply:
That was for lint to work. Also, we already shipped v4.0.1 without any problems so I wouldn't worry about it.

Reply:
Ohhhhhhhhh, but I forgot that it will make it hard for people to use the sample 😕. Well unless we want to undo a few weeks of PRs, there's no going back. Sorry about that! 😊

Reply:
NP let's just get onto a beta-or-better train as soon as one is available.